### PR TITLE
Task/not propage unprovisioned config from group to device

### PR DESCRIPTION
**Pull request by AlvaroVega**
reopens https://github.com/telefonicaid/iotagent-node-lib/pull/1391 to include explicitAttrs
No CNR Update is needed

### CODE REVIEW

File: lib/services/devices/deviceService.js [hunk 1]
```diff
@@ -181,9 +181,6 @@ function mergeDeviceWithConfiguration(fields, defaults, deviceData, configuratio
     if (configuration && configuration.expressionLanguage && deviceData.expressionLanguage === undefined) {
         deviceData.expressionLanguage = configuration.expressionLanguage;
     }
```
Comment (fgalan) [inline lib/services/devices/deviceService.js:183]:
```suggestion
```

I understand this should be also removed.

(Maybe a grep search by `expressionLanguage` could help to identify other similar leftover we may have yet)

Comment (mapedraza) [reply lib/services/devices/deviceService.js:183]: Related https://github.com/telefonicaid/iotagent-node-lib/issues/1384

Comment (AlvaroVega) [reply lib/services/devices/deviceService.js:183]: IMHO Delete of expressionLanguague should be done in another PR, with all related stuff https://github.com/telefonicaid/iotagent-node-lib/issues/1384

Comment (AlvaroVega) [reply lib/services/devices/deviceService.js:183]: > IMHO Delete of expressionLanguague should be done in another PR, with all related stuff #1384

Done in https://github.com/telefonicaid/iotagent-node-lib/issues/1384

**Review by fgalan — APPROVED** [review]
LGTM

**Review by mapedraza — APPROVED** [review]
LGTM

**Pull request merged**